website/views/shared/modals/quests.jade

```diff
@@ -90,3 +77,17 @@ script(type='text/ng-template', id='modals/questInvitation.html')
     button.btn.btn-default(ng-click='questHold = true; $close()')=env.t('askLater')
     button.btn.btn-default(ng-click='questReject(party); $close()')=env.t('reject')
     button.btn.btn-primary(ng-click='questAccept(party); $close()')=env.t('accept')
+
+script(type='text/ng-template', id='modals/questDrop.html')
+  .quest-icon.pull-right(class='inventory_quest_scroll_{{::selectedQuest.key}}')
+  .modal-header
+    h4=env.t('messageDropQuest')
+  .modal-body
+    h4 {{::selectedQuest.text()}}
+    +questInfo
+    hr
+    .npc_ian.pull-left
+    p=env.t('dropQuestCongrats')
+  .modal-footer
+    button.btn.btn-default(ng-click='closeQuest(); $close()')=env.t('questLater')
+    button.btn.btn-primary(ng-click='questInit(); $close()')=env.t('inviteParty')
```

Comment:
I wanted to have this conditionally be the 'create a party` button, but `inviteOrStartParty` wasn't working here.